test/parallel/test-repl-built-in-modules.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const common = require('../common');
+const tmpdir = require('../common/tmpdir');
+const assert = require('assert');
+const spawn = require('child_process').spawn;
+
+tmpdir.refresh();
+
+function run(wasiPreview) {
+  let resolve;
+  const promise = new Promise((_resolve) => {
+    resolve = _resolve;
+  });
+
+  // Use -i to force node into interactive mode, despite stdout not being a TTY
+  const args = ['-i'];
+  if (wasiPreview) args.push('--experimental-wasi-unstable-preview1');
+  const child = spawn(process.execPath, args);
+
+  const input = 'require(\'events\');\nrequire(\'wasi\');';
+  let stdout = '';
+  let stderr = '';
+
+  child.stdout.setEncoding('utf8');
+  child.stdout.on('data', (c) => stdout += c);
+  child.stderr.setEncoding('utf8');
+  child.stderr.on('data', (c) => stderr += c);
+
+  child.stdin.end(input);
+
+  child.on('exit', () => {
+    resolve({ stdout, stderr });
+  });
+
+  return promise;
+}
+
+(async function() {
+  const ret1 = await run(false);
+  const ret2 = await run(true);
+
+  assert(/\[Function: EventEmitter\] {/.test(ret1.stdout));
```

Comment:
```suggestion
  assert.match(ret1.stdout, /\[Function: EventEmitter\] {/);
```

Same for the others.